sdk/core/amqp-common/changelog.md

```diff
@@ -1,3 +1,8 @@
+### 2020-03-13 1.0.0-preview.11
+
+- Increase the default timeout for the requests sent to the service from 10s to 60s to avoid `"$cbs" timeout` during the CBS claim negotiation.
```

Comment:
Use back ticks for code only

```suggestion
- Increase the default timeout for the authorization requests sent to the service from 10s to 60s to avoid frequent timeout errors during the CBS claim negotiation.
```